mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala

```diff
@@ -338,6 +356,12 @@ final class ActorMqttServerSession(settings: MqttSessionSettings)(implicit mat:
 
   import system.dispatcher
 
+  override def ![A](cp: Command[A]): Unit = cp match {
+    case Command(cp: Publish, carry) =>
+      serverConnector ! ServerConnector.PublishReceivedLocally(cp, carry)
+    case c: Command[_] => throw new IllegalStateException(c + " is not a server command that can be sent directly")
```

Comment:
Is there benefit in allowing `Subscribe`, `Unsubscribe` here as well? Or perhaps as it is now, resending those remains the client's responsibility?

Reply:
I think those ones should remain the client responsibility. They are idempotent and the session isn’t persistent about their retries ie you really should have a connection established.